test/unit/apiRequest.js

```diff
@@ -4,11 +4,13 @@ const apiRequest = require('../../app/lib/apiRequest');
 const expect = chai.expect;
 
 describe('apiRequest', () => {
-  it('should error if the status code is not 2xx', () =>
+  it('should error if the status code is not 2xx', function test() {
+    this.timeout = 10000;
```

Comment:
Is it worth a comment about not using an arrow function because of the need to bind to `this`?